soccer/src/soccer/planning/planner/plan_request.cpp

```diff
@@ -2,43 +2,62 @@
 
 namespace planning {
 
+std::shared_ptr<rj_geometry::Circle> calc_static_robot_obs(const RobotState& robot) {
+    // params for obstacle shift
+    double obs_center_shift = 0.5;
+    double obs_radius_inflation = 0.5;
```

Comment:
```suggestion
    double obs_radius_inflation = 1;
```
One change I played around with which slightly helped reduce number of collisions